src/System.Private.CoreLib/shared/System/Diagnostics/SymbolStore/ISymbolDocumentWriter.cs

```diff
@@ -2,11 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
 namespace System.Diagnostics.SymbolStore
 {
     public interface ISymbolDocumentWriter
     {
-        void SetCheckSum(Guid algorithmId, byte[] checkSum);
-        void SetSource(byte[] source);
+        void SetCheckSum(Guid algorithmId, byte?[]? checkSum);
```

Comment:
Why is this nullable?  Are there implementations that expect null?  The one you included in this PR will null ref is null is passed for `checkSum`.